makefiles/Test.mk

```diff
@@ -54,7 +54,7 @@ audit:
 	go vet ./...
 	go run honnef.co/go/tools/cmd/staticcheck@latest -checks=all,-ST1000,-U1000 ./...
 	go run golang.org/x/vuln/cmd/govulncheck@latest ./...
-	go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run
+	CGO_ENABLED=0 go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run
```

Comment:
Not using this had issues in Fedora.

It's probably some missing gcc binaries or something, but since we don't use CGO in lints anyway, it does not hurt to explicitly disable it.